### DISCUSSION TITLE
Hello from a new developer!

### DISCUSSION THREAD

**Discussion by pongo-pygmaeus** [ACINQ/eclair#2062] [Developers]
Hi everyone,

I'm a software engineer in the SF Bay Area looking to help out with an L2 BTC project. I've been watching Eclair's development for a while. I've written web APIs in Scala with Play for the past 4 years and would be interested in taking on some of the open `Good First Issue` tickets. Do you folks still need help with these issues?

https://github.com/ACINQ/eclair/issues/1858
https://github.com/ACINQ/eclair/issues/1616
https://github.com/ACINQ/eclair/issues/1741

Best,
Max

**Comment by t-bast** [comment]
Hey Max!

Sure, it would be great to get these fixed, and it's a good way for you to get a feel for some parts of the codebase.
Don't hesitate to share ideas / questions on the issues as you progress, we will help you throughout this process.

**Comment by pongo-pygmaeus** [comment]
Thanks, Bastien! That all sounds great. I'll get started as soon as I can.
